pytmx/pytmx.py

```diff
@@ -1451,23 +1450,28 @@ class TiledObjectGroup(TiledElement, list):
 
     """
 
-    def __init__(self, parent, node, customs) -> None:
+    def __init__(
+        self,
+        parent: TiledMap,
+        node: ElementTree.Element,
+        customs: Optional[dict] = None,
+    ) -> None:
         TiledElement.__init__(self)
         self.parent = parent
 
         # defaults from the specification
         self.name = None
         self.color = None
         self.opacity = 1
-        self.visible = 1
+        self.visible = True
```

Comment:
before **visible** was a mix of (int, boolean and string), same doubt, 1 is True?